src/main/webapp/mapScript.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Initialize the map to show Mountain View and events in that area
+ */
+function initMap() {
+
+  // Coordinates for the default map center of Mountain View, CA
+  const mapViewDefault = {lat: 37.3868, lng: -122.085}; 
+
+  const map = new google.maps.Map(document.getElementById('map'), {
+    zoom: 8,
+    center: mapViewDefault,
+  });
+  
+  const geocoder = new google.maps.Geocoder();
```

Comment:
if geocoder's only being used once, you can just pass in `new Geocoder()` to geocodeAddress()